src/textual/dom.py

```diff
@@ -278,6 +278,10 @@ def add_children(tree, node):
         add_children(tree, self)
         return tree
 
+    @property
+    def displayed_children(self) -> NodeList:
```

Comment:
I don't think this needs to be a `NodeList`, which is used to control how the dev adds widgets. Plain old List is fine.